src/main/java/uk/gov/mca/beacons/api/jpa/entities/BeaconNote.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.mca.beacons.api.jpa.entities;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.validation.constraints.Email;
+import lombok.Getter;
+import lombok.Setter;
+import uk.gov.mca.beacons.api.domain.NoteType;
+
+@Entity
+@Getter
+@Setter
+public class BeaconNote {
+
+  @Id
+  @GeneratedValue
+  private UUID id;
+
+  private UUID beaconId;
+
+  private String note;
+
+  @Enumerated(EnumType.STRING)
+  private NoteType type;
```

Comment:
Is `type` okay, or would `noteType` be better here like `personType`? 
Thought `BeaconNote.type` might be nicer than `BeaconNote.noteType` 🤷

Reply:
I think `type` is okay? It's in the context of a `Note` so I don't think you need the prefix of `note`